iac-extensions/docker/src/test/resources/checks/SecretsGenerationCheck/sshKeygen.dockerfile

```diff
@@ -0,0 +1,109 @@
+FROM ubuntu:22.04
+
+# Noncompliant@+1 {{Revoke and change this secret, as it might be compromised.}}
+RUN ssh-keygen -N "" -t dsa -b 1024 -f rsync-key
+#   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
```

Comment:
I would add this test case (not necessarily here) :
`RUN ssh-keygen -N "" -t dsa -N "val" -b 1024 -f rsync-key`